src/Interface/Modules/Render/ES/SRInterface.cc

```diff
@@ -828,7 +828,7 @@ uint32_t SRInterface::getSelectIDForName(const std::string& name)
         * (glm::length(originToCurrentSpos) / glm::length(originToSpos_));
 
       // Flip if negative to avoid inverted normals
-      glm::mat4 flip;
```

Comment:
Same with this one, make it `SCIMat4`

Reply:
It can be in a separate PR if you want

Reply:
GLMMat4?

Reply:
SCI or GLM are pretty redundant. How about highlighting the use case (https://en.wikipedia.org/wiki/Transformation_matrix), so TransformationMatrix or ProjectiveMatrix depending on the context?